scripts/build-circt.sh

```diff
@@ -43,12 +48,15 @@ while [[ "$#" -ge 1 ]] ; do
 	esac
 done
 
-git clone https://github.com/EECS-NTNU/circt.git ${CIRCT_BUILD}
-cd ${CIRCT_BUILD}
+if [ ! -d "$CIRCT_BUILD/circt.git" ] ;
+then
+	git clone https://github.com/EECS-NTNU/circt.git ${CIRCT_BUILD}/circt.git
+fi
+cd ${CIRCT_BUILD}/circt.git
```

Comment:
Keeping this line makes it possible to use conventional git syntax instead of relying on git commands that is likely not used by many and adds complexity.